copilot/fsd-assessment-store/manifest.yml

```diff
@@ -57,6 +57,8 @@ environments:
   test:
     deployment:
       rolling: 'recreate'
+    cpu: 1024
+    memory: 2048
```

Comment:
Does it make sense to be running with more resources in test than in production? Should we change the default for the app instead?

Reply:
The thinking was that test runs e2e testing frequently.  UAT and Prod are the same.  I'm happy to change them all to be higher if that seems better.

Reply:
Dev: 
![image](https://github.com/communitiesuk/funding-service-design-assessment-store/assets/125875751/f2241a13-91cb-42b9-aedd-381ccc317587)


Test:
![image](https://github.com/communitiesuk/funding-service-design-assessment-store/assets/125875751/7fec7b8c-fa7e-49a3-afce-78f25de7338e)


UAT:
![image](https://github.com/communitiesuk/funding-service-design-assessment-store/assets/125875751/79d5ad3f-3ccf-4838-a7cb-1ec96ccb8b83)


Prod:
![image](https://github.com/communitiesuk/funding-service-design-assessment-store/assets/125875751/156f4719-76a6-455b-a349-ba4dd6537952)

Reply:
Test is regularly pushing the limits of it's resources, I suspect due to the E2E tests that run on assessment

Reply:
Seems a bit weird, e.g. if we are running performance tests on pre-prod environments it will behave differently on production, so I'd rather level up the default for that reason.

Reply:
This is on the test environment, which will, by it's very nature, be stressed MORE than production environments. Because you're running tests against it, it will have more connections, use more resources, stress the CPU more often, etc. Development may end up this way as well, depending on how we end up architecting the dev environment, but the stress of this is usually taken away by running all of that functionality locally. That's why it goes test -> uat -> prod

Reply:
If we level up the default, in my opinion we're wasting resources on environments that don't need buffing, but I'm not fussed either way - we can either up the default, or test specifically based on the load patterns

Reply:
For now, we'll get this in.  Resolving conversation.  If required we can extend this increase to the other envs.